File: math/Java/factorial.java
//Atieve Wadhwa
//Finding factorial of a number without recursion
import java.util.*;

public class factorial
{
	public static void main(String args[])
	{
		System.out.print("Enter Number: ");
		Scanner s = new Scanner(System.in);
		int n = s.nextInt();
		int ans = 1;
		while(n>1)
		{
			ans = n*ans;
			n = n-1;
		}

		System.out.println("The factorial is: " + ans);
		s.close();
	}
}

File: math/C++/factorial.cpp
//program to find factorial of a given no. without using recursive function.

#include <iostream>
using namespace std;
int fac(int n){
 int m=1;
    for(int j=1; j<=n; j++)
    {
        m=m*j;
    }
    return m;
}

int main(){
int i,n;
cout<<"enter the no. for finding the factorial : \n";
cin>>n;
i=fac(n);
cout<<"the factorial of the given no. is "<<i;
}
